src/test/util/setup_common.cpp

```diff
@@ -195,7 +195,7 @@ TestingSetup::TestingSetup(const std::string& chainName, const std::vector<const
     }
 
     m_node.addrman = std::make_unique<CAddrMan>();
-    m_node.banman = std::make_unique<BanMan>(m_args.GetDataDirPath() / "banlist.dat", nullptr, DEFAULT_MISBEHAVING_BANTIME);
+    m_node.banman = std::make_unique<BanMan>(m_args.GetDataDirBase() / "banlist.dat", nullptr, DEFAULT_MISBEHAVING_BANTIME);
```

Comment:
In commit "Use `ArgsManager.GetDataDirBasePath()` and `ArgsManager.GetDataDirNetPath()` instead of `ArgsManager.GetDataDirPath(net_identifier)`" (80dec1717258fd74ae32274affed71daf2949e46)

IMO it would be a little better to keep the `GetDataDirPath` method and argumentless `GetDataDirPath()` calls unchanged in this commit, so the commit is smaller. Then the subsequent scripted-diff commit could do mechanical work of replacing `m_args.GetDataDirPath()` with `m_args.GetDataDirNet()` and `GetDataDirPath` with `GetDataDir`.

Also, it doesn't matter since it only affects tests, but replacing `GetDataDirPath()` with `GetDataDirNet()` instead of `GetDataDirBase()` would be a more obvious choice to keep behavior unchanged.

This is all fine, though. Just extra suggestions in case there's a reason to update.

Reply:
I have moved the tests changes to a separate commit (https://github.com/bitcoin/bitcoin/pull/21850/commits/7f5774c09affebc8837a4d117aae198d6fd6157a). 

The PR differs in one trailing newline as an effect. So there is no code change, only git commit changes.